Jetstream/Session.swift

```diff
@@ -108,7 +107,7 @@ public class Session {
         
         switch message {
         case let scopeStateMessage as ScopeStateMessage:
-            if let scope = scopes[scopeStateMessage.scopeIndex] {
+            if let scope = scopes[scopeStateMessage.scopeIndex]?.0 {
```

Comment:
If your tuple variables have name you don't need to use 0, but can use its name.